src/main/java/fi/otavanopisto/kuntaapi/server/index/IndexUpdater.java

```diff
@@ -0,0 +1,118 @@
+package fi.otavanopisto.kuntaapi.server.index;
+
+import java.beans.BeanInfo;
+import java.beans.IntrospectionException;
+import java.beans.Introspector;
+import java.beans.PropertyDescriptor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.PostConstruct;
+import javax.ejb.Lock;
+import javax.ejb.LockType;
+import javax.ejb.Singleton;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@ApplicationScoped
+@Singleton
+public class IndexUpdater extends AbstractIndexHander {
+  
+  @Inject
+  private Logger logger;
+  
+  @PostConstruct
+  public void initIndexables() {
+    registerIndexable(IndexableOrganization.class);
+    registerIndexable(IndexableService.class);
+    registerIndexable(IndexablePage.class);
+  }
+
+  @Lock (LockType.READ)
+  public void index(Indexable indexable) {
+    getClient().prepareIndex(getIndex(), indexable.getType(), indexable.getId())
+      .setSource(serialize(indexable))
+      .execute()
+      .actionGet();
+  }
+  
+  private void registerIndexable(Class<? extends Indexable> indexable) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "registerIndexable" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)